packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -192,8 +192,6 @@ const PostsPagePostHeader = ({post, answers = [], toggleEmbeddedPlayer, hideMenu
     <AnalyticsContext pageSectionContext="topSequenceNavigation">
       <PostsTopSequencesNav post={post} />
     </AnalyticsContext>
-    {!post.group && !post.sequence && !post.question && <PostsPageTopTag post={post} />}
```

Comment:
Are we just getting rid of this? Was this in the designs or did you just infer it?

Reply:
Yep - it's in the designs 🙂